index.html

```diff
@@ -1,177 +1,238 @@
-          <div class="project-card">
-            <div class="project-img">
-              <img src="Images/Third.png" alt="a brief description of what we do">
-            </div>
-            <div class="project-title">
-              <h2>Tonic</h2>
-            </div><div class="info">
-              <span class="canopy">CANOPY</span> <img src="./Images/Counter.png" alt="dot">
-              <span class="">back end dev</span> <img src="./Images/Counter.png" alt="dot">
-             <span>2015</span>
-            </div >
-            <div class="project-paragraph">
-              <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
-            </div>
-              <ul class="technologies">
-                <li>html</li>
-                <li>css</li>
-                <li>javaScript</li>
-              </ul>
-              <button type="button" class="project-btn">see project</button>
+      <section class="introduction">
+        <div>
+          <h1 class="intro-text">I’m Butler Glad to see you!</h1>
+          <p class="text-2">
+            I’m a software developer! I can help you build a product , feature
+            or website Look through some of my work and experience! If you like
+            what you see and have a project you need coded, don’t hestiate to
+            contact me.
+          </p>
         </div>
-        <!--fourth card-->
-        <div class="project-card">
+        <div>
+          <h2 class="icon-text">LET'S CONNECT</h2>
+          <ul class="social-icons">
+            <li>
+              <a href="#"><i class="fa fa-twitter" aria-hidden="true"></i></a>
+            </li>
+            <li>
+              <a href="#"><i class="fa fa-linkedin" aria-hidden="true"></i></a>
+            </li>
+            <li>
+              <a href="#"><i class="fa fa-medium" aria-hidden="true"></i></a>
+            </li>
+            <li>
+              <a href="#"><i class="fa fa-github" aria-hidden="true"></i></a>
+            </li>
+            <li>
+              <a href="#"><i class="fa fa-angellist" aria-hidden="true"></i></a>
+            </li>
+          </ul>
+        </div>
+      </section>
+      <section class="works">
+        <!--First card-->
+        <article class="project-card">
           <div class="project-img">
-            <img src="Images/Fourth.png" alt="a brief description of what we do">
+            <img
+              src="Images/First.png"
+              alt="a brief description of what we do"
+            />
           </div>
-          <div class="project-title">
-            <h2>Multi-post stories</h2>
+
+          <h2 class="project-title">Tonic</h2>
+
+          <div class="info">
+            <span class="canopy">CANOPY</span>
+            <img src="./Images/Counter.png" alt="dot" />
+            <span class="">back end dev</span>
+            <img src="./Images/Counter.png" alt="dot" />
+            <span>2015</span>
+          </div>
```

Comment:
The `div` with the info class has the following styling from inspection.
```
.info {
    margin: 0 auto;
}
```
The applied style aligns the entire element in the center.  However, in the figma style this element is aligned to the left.

| Your design | Figma design |
| --- | --- |
| <img width="286" alt="fuu2" src="https://user-images.githubusercontent.com/4247740/143687030-de240e5a-daa1-49cc-b8b1-9dd5c43953db.PNG"> | <img width="247" alt="fuu" src="https://user-images.githubusercontent.com/4247740/143687007-6797c1dc-d667-46b0-b183-13819c065c8e.PNG"> |

- [ ] I suggest that you remove that style to normalize the alignment.

Reply:
- Done.